Measurements/Measurements.ipynb

```diff
@@ -32,14 +32,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%package Microsoft.Quantum.Katas::0.12.20082513"
+    "%package"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "> The package versions in the output of the cell above should always match. If you are running the Notebooks locally and the versions do not match, please install the IQ# version that matches the version of the `Microsoft.Quantum.Katas` package.\n",
+    "> The package versions in the output of the cell above should always match the latest version which is `Microsoft.Quantum.Katas::0.12.20082513`. If you are running the Notebooks locally and the versions do not match, please install the IQ# version that matches the version of the `Microsoft.Quantum.Katas` package.\n",
     "> <details>\n",
     "> <summary><u>How to install the right IQ# version</u></summary>\n",
     "> For example, if the version of `Microsoft.Quantum.Katas` package above is 0.1.2.3, the installation steps are as follows:\n",
```

Comment:
Do you want to remove the "0.1.2.3" from here as well?

Reply:
I have updated this to include the actual version.